tests/unit/modules/test_win_file.py

```diff
@@ -324,6 +320,8 @@ def test_check_perms_reset(self):
             reset=True)
         self.assertDictEqual(expected, ret)
 
+    @destructiveTest
```

Comment:
If the symlink lives on a temporary directory the test suite created then this is not a destructive test.

Reply:
Sweet. I thought it was any modification to the system

Reply:
We're free to destroy whatever temp files/dirs we create :)

Reply:
So, those were being created in the user temp directory. I modified all tests in the module to use Salt's temp directory and removed the destructive decorator